src/components/Index/ContributorsSection.tsx

```diff
@@ -1,115 +1,131 @@
+                            x => x.node.name === author.photo
+                          ).node.childImageSharp.gatsbyImageData
+                        }
+                        className="rounded-full overflow-hidden gatsby-image-wrapper-rounded"
+                        alt={author.name}
+                        style={{ width: '100%', height: '100%' }}
+                      />
+                    </div>
+                    <div className="space-y-1">
+                      <div className="text-sm font-medium lg:text-sm">
+                        <a
+                          href={`https://github.com/${author.githubUsername}`}
+                          target="_blank"
+                          rel="noreferrer"
+                          className="text-blue-600 dark:text-blue-400"
+                        >
+                          <h3>{author.name}</h3>
+                        </a>{' '}
+                      </div>
+                      <p className="text-gray-500 dark:text-gray-400 list-disc list-inside text-xs space-y-2 mb-6">
+                        Since {author.sinceDate}
+                      </p>
+                    </div>
+                  </div>
+                </li>
+              ))}
+            </ul>
+
+            <h1 className="mt-2 mb-8 text-3xl text-center leading-8 font-extrabold tracking-tight text-gray-900 dark:text-gray-100 sm:text-xl sm:leading-10">
+              Past Content Authors:
+            </h1>
+            <ul className="mx-auto grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-4 md:gap-x-6 lg:max-w-5xl lg:gap-x-8 lg:gap-y-12 xl:grid-cols-6">
+              {pastContentAuthors.map(author => (
+                <li key={author.name}>
+                  <div className="space-y-2">
+                    <div className="w-36 h-36 sm:h-28 sm:w-28 md:h-36 md:w-36 xl:h-32 xl:w-32 mx-auto">
+                      <GatsbyImage
+                        image={
+                          (data as any).allFile.edges.find(
+                            x => x.node.name === author.photo
+                          ).node.childImageSharp.gatsbyImageData
+                        }
+                        className="rounded-full overflow-hidden gatsby-image-wrapper-rounded"
+                        alt={author.name}
+                        style={{ width: '100%', height: '100%' }}
+                      />
+                    </div>
+                    <div className="space-y-2">
+                      <div className="text-sm font-medium lg:text-sm">
+                        <a
+                          href={`https://github.com/${author.githubUsername}`}
+                          target="_blank"
+                          rel="noreferrer"
+                          className="text-blue-600 dark:text-blue-400"
+                        >
+                          <h3>{author.name}</h3>
+                        </a>{' '}
+                      </div>
+                    </div>
```

Comment:
you don't need the outer div because there is no "Since [date]" text, meaning you don't need to control spacing between the "name" and "since" text. if it causes an error (hopefully it doesn't), u can revert the change

```suggestion
                    <div className="text-sm font-medium lg:text-sm">
                      <a
                        href={`https://github.com/${author.githubUsername}`}
                        target="_blank"
                        rel="noreferrer"
                        className="text-blue-600 dark:text-blue-400"
                      >
                        <h3>{author.name}</h3>
                      </a>{' '}
                    </div>
```